### PR TITLE
Dev --> Staging v4.0.0 release

### PR DESCRIPTION
**Pull request by cacheung**
Dev --> Staging v4.0.0 release

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ let package = Package(
         .library(name: "AEPEdgeBridge", targets: ["AEPEdgeBridge"])
```
Comment (addb) [inline Package.swift:20]: L18 
update platforms to `.iOS(.v11)`

File: Documentation/tutorials/EdgeBridgeTutorialAppStart/EdgeBridgeTutorialApp.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      "identity" : "aepsdk-core-ios",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/adobe/aepsdk-core-ios.git",
+      "state" : {
+        "branch" : "main",
+        "revision" : "255e3fde7c478a48c57e3f6ee9df1700aae22278"
+      }
+    },
+    {
+      "identity" : "aepsdk-edge-ios",
```
Comment (emdobrin) [inline Documentation/tutorials/EdgeBridgeTutorialAppStart/EdgeBridgeTutorialApp.xcworkspace/xcshareddata/swiftpm/Package.resolved:31]: I think the start app should not have these dependencies yet, right? only after the tutorial steps you will add them

Comment (emdobrin) [reply Documentation/tutorials/EdgeBridgeTutorialAppStart/EdgeBridgeTutorialApp.xcworkspace/xcshareddata/swiftpm/Package.resolved:31]: nvm, after reading more through I think this is correct for the start app, you just add edgrebridge later

Comment (cacheung) [reply Documentation/tutorials/EdgeBridgeTutorialAppStart/EdgeBridgeTutorialApp.xcworkspace/xcshareddata/swiftpm/Package.resolved:31]: Should I keep the resolved file there?

File: Package.swift (hunk 1 above)
Comment (addb) [inline Package.swift:20]: platforms: [.iOS(.v11)]

**Review by addb — APPROVED** [review]

**Pull request merged**